static/js/benchmark.js

```diff
@@ -126,11 +126,25 @@
 
   // fix navigation header to top after scroll 
   $(window).scroll(function() {
-      if ($(this).scrollTop() > headerPosition) {
+
+      var scrollPosition = $(this).scrollTop();
+      var margin = 150;
+      if ( scrollPosition > headerPosition) {
           $('.cr-nav').addClass("sticky");
       } else {
           $('.cr-nav').removeClass("sticky");
       }
+
+      $('.cr-nav .cr-nav-item').each(function(){
+       var refElement = $(this).attr('scroll-to-chart');
+       if($('#'+refElement +'_tab').position().top - margin <= scrollPosition && $('#'+refElement +'_tab').position().top + $('#'+refElement +'_tab').height() > scrollPosition + margin){
```

Comment:
code style: try to make it more readable (split up into multiple lines, new variables)